x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/description_step/helpers.tsx

```diff
@@ -167,11 +171,17 @@ export const buildUnorderedListArrayDescription = (
       {
         title: label,
         description: (
-          <ul>
-            {values.map((val: string) =>
-              isEmpty(val) ? null : <li key={`${field}-${val}`}>{val}</li>
-            )}
-          </ul>
+          <EuiText size="s">
+            <ul>
+              {values.map((val: string) =>
```

Comment:
You don't need the extra typing here. You can just use `(val) =>`

You can check in your IDE for this. Redundant typing is usually looked upon as just being noisy. Up to you if you want to change, just mentioning it as it's considered good "ergonomics"/"habits" to remove redundant noise.

<img width="667" alt="Screen Shot 2020-03-13 at 5 17 22 PM" src="https://user-images.githubusercontent.com/1151048/76667325-c8dcf600-654e-11ea-8cc2-db4ac0387e70.png">

Reply:
Ah, I see the redundancy there - makes sense. Thanks for pointing this out. I went ahead and removed it.